src/extension/ncwms/src/main/java/au/org/emii/geoserver/wms/Ncwms.java

```diff
@@ -31,16 +33,19 @@
          <wfsServer>http://localhost:8080/geoserver/ows</wfsServer>
          <urlSubstitution key="/mnt/imos-t3/IMOS/opendap/">http://thredds-1-aws-syd.aodn.org.au/thredds/wms/IMOS/</urlSubstitution>
          <urlSubstitution key="^/IMOS/">http://thredds-1-aws-syd.aodn.org.au/thredds/wms/IMOS/</urlSubstitution>
+         <collectionsWithTimeMismatch>^imos:srs.*</collectionsWithTimeMismatch>
```

Comment:
I leant in favour of making it clear why the timefield was being disregarded rather than just saying to disregard it - I'd prefer to remove this code altogether and am pushing to reharvest all the data with the correct timestamps